scripts/zones/La_Theine_Plateau/npcs/Vicorpasse.lua

```diff
@@ -27,7 +27,7 @@ function onTrigger(player,npc)
 	if(player:getCurrentMission(SANDORIA) == THE_RESCUE_DRILL) then
 		local MissionStatus = player:getVar("MissionStatus");
 		
-		if(MissionStatus == 4) then
+		if(MissionStatus >= 0 and MissionStatus <= 4) then

```

Comment:
Everywhere online says this is incorrect. And I remember many times on retail accidentally missing him and having to go back up that hill to click him, then back down to the other NPCs in order to proceed.